src/app/history/recorder.js

```diff
@@ -84,6 +84,9 @@ export default class Recorder {
   }
 
   getPathFor(serviceId, documentType, fileExtension) {
+    if (serviceId === null) {
```

Comment:
When no service provider is specified, documents are stored at the root of the repo.